src/reporters/lang/en.js

```diff
@@ -88,6 +88,7 @@ const messages = {
   jsonError: 'Error parsing JSON at $0, $1.',
   noFilePermission: "We don't have permissions to touch the file $0.",
   allDependenciesUpToDate: 'All of your dependencies are up to date.',
+  frozenLockfileError: 'Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`',
```

Comment:
[The past participle of run is run.](https://www.quora.com/Which-is-more-grammatically-correct-was-run-or-was-ran) I will add the period.